joomla/joomla-cms
# [5.4] Schema.org: Fix error when menu item doesn't exist

### Summary of Changes
When having the schema.org plugin enabled and opening a URL with a non-existing Itemid, the plugin throws a fatal error.


### Testing Instructions
1. Open a URL like `index.php?Itemid=666&option=com_content&view=category&id=2`
2. See the fatal error
3. Apply patch
4. See error go away.


### Link to documentations
Please select:
